File: Libraries/ClassLibrary/DTO/BikeDTO.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ClassLibrary.DTO
{
    public class BikeDTO
    {
        public string Id { get; set; }
        public StationDTO Station { get; set; }
        public UserDTO User { get; set; }
        public BikeStatusDTO BikeStatus { get; set; }
    }
}

File: Frontend/UserTech/src/app/dto/bike-dto.ts
import {StationDTO} from './station-dto';
import {UserDTO} from './user-dto';
import {BikeState} from '../models/bike';

export interface BikeDTO {
  id: string;
  station: StationDTO | null;
  user: UserDTO | null;
  bikeStatus: BikeState;
}
